### PR TITLE
vaspacx quiz UI 

### PR DESCRIPTION
**Pull request by vashnavichauhan1825**
I added dummy data for questions and categories

### CODE REVIEW

**Comment by vashnavichauhan1825** [comment]
live url:-https://vaspacx-quiz-react.netlify.app/

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+import {Route,Routes} from 'react-router-dom'
+import CategoryPage from "./pages/category/CategoryPage";
+import QuePage from "./pages/quePage/QuePage";
+import ResultPage from "./pages/result/ResultPage";
+import CategoryRules from "./components/Rules/CategoryRules";
+import Quiz from "./components/quizes/Quiz";
+import { lightTheme } from "./components/style/stylecomponents/themes";
+import { darkTheme } from "./components/style/stylecomponents/themes";
+import { useSelector } from "react-redux";
 function App() {
```
Comment (ashishkatiyar2404) [inline src/App.js:13]: always give one line space b/n import files and function

**Review by ashishkatiyar2404 — APPROVED** [review]
Code looks clean and functionality is also good. 
UI is 🔥. 
you  can merge and continue working on  other files.

File: src/App.js (hunk 1 above)
Comment (vashnavichauhan1825) [reply src/App.js:13]: ok sir🚀

**Pull request merged**